docs/howtos/install_epinio_on_rke.md

```diff
@@ -1,35 +1,78 @@
 ---
-sidebar_label: "Installing Epinio On RKE2 (Rancher)"
-sidebar_position: 18
+sidebar_label: "Installing Epinio On RKE2"
+sidebar_position: 19
 title: ""
 ---
 
-# Rancher RKE2 configuration
+# Creating an RKE2 Kubernetes Cluster
+This guide will help you to deploy a suitable RKE2 Kubernetes cluster for Epinio. More details can be found in RKE2 [quickstart](https://docs.rke2.io/install/quickstart/) guide.
 
 This page was written using the following versions:
-* [epinio helm chart 0.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-0.7.1)
-* RKE2 version v1.22.7+rke2r2
+* [epinio helm chart 1.8.0](https://github.com/epinio/helm-charts/releases/tag/epinio-1.8.0)
+* RKE2 version v1.25.9+rke2r1
+* openSUSE Leap 15.4
```

Comment:
I think we should avoid hardcoding specific releases, and let the users pick the latest one. Maybe these information can be added at the very end of the page, just as a reference, but otherwise we will maybe end up with users trying to install very old versions of Epinio, or we need to update this page everytime.

Reply:
indeed kind of useless info, removed via 7e9d308e73ba58bd0afbf468a92d7f9ccbfe5533